scripts/game_structure/windows.py

```diff
@@ -414,9 +414,9 @@ def process_event(self, event):
                                                     use_prefix,
                                                     None,
                                                     self.the_cat.pelt.colour,
-                                                    self.the_cat.eye_colour,
```

Comment:
There is also some bits above this section that should be changes to fix the error (adding pelt. in front of it.)